tests/test_identifier.py

```diff
@@ -4,5 +4,14 @@
 def test_identifier_works():
     r = identifier.Identifier()
     out = r.identify("DANHz6EQVoWyZ9rER56DwTXHWUxfkv9k2o")
-    assert "Dogecoin (DOGE) Wallet Address" in out["Regexes"][0]["Regex Pattern"]["Name"]
+    assert "Dogecoin (DOGE) Wallet Address" in out["Regexes"]["None"][0]["Regex Pattern"]["Name"]

```

Comment:
Why is `"None"` added here? This looks like it'll break the API, everyone will have to update their own scripts -- no?

Reply:
This value is a filename, but if there is no file, I made to be None (that obviously was not the best idea) so that it could be easily printed in the table as None if there is no file. Will change this to "text" and when printing change it to "None".